src/dbus/abrt_problems2_service.h

```diff
@@ -86,7 +86,7 @@ GVariant *abrt_p2_service_entry_problem_data(AbrtP2Service *service,
             GError **error);
 
 
-enum {
+extern enum {
```

Comment:
We don’t need this at all. A better thing to do would be to `typedef` it and rename to `AbrtP2ServiceEntryLookupFlags` or something like that and then use that instead of plain `int` in parameters/variables.

Reply:
Changed.